test/runtests.jl

```diff
@@ -40,6 +54,28 @@ using Test
                     Map(_.y)
 
     @test [1] == @_(Map(_.y) ∘ Filter(startswith(_.x, "a")))(data)
+
```

Comment:
Good examples of things which should work :+1:

I must admit some of these caught me by surprise, mostly in a good way :-) Taken out of context they look like things one "shouldn't" do for readability (that broadcast example does my head in!) or genericity (eg, accessing `.re`).

I may add a small note that not all of these things are recommended as isolated examples but that they could make sense in the context of a larger data pipeline.